lighthouse-core/gather/driver.js

```diff
@@ -433,12 +433,14 @@ class Driver {
    *    See https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query.
    */
   queryPermissionState(name) {
+    if (['geolocation', 'notifications', 'midi', 'push'].indexOf(name) === -1) {
```

Comment:
I'm not too concerned about protecting against all errors like this. Shouldn't `exceptionDetails`/when we land #1037 catch syntax errors like that?